build/Dockerfile

```diff
@@ -35,3 +35,9 @@ RUN mkdir -p /var/lib/kepler/data
 COPY --from=builder /opt/app-root/src/github.com/sustainable-computing-io/kepler/data/normalized_cpu_arch.csv /var/lib/kepler/data/normalized_cpu_arch.csv
 
 ENTRYPOINT ["/usr/bin/kepler"]
+
+# from https://raw.githubusercontent.com/sustainable-computing-io/kepler-model-server/main/tests/test_models/DynComponentModelWeight/CgroupOnly/ScikitMixed/ScikitMixed.json
+COPY --from=builder /opt/app-root/src/github.com/sustainable-computing-io/kepler/data/ScikitMixed.json /var/lib/kepler/data/ScikitMixed.json
+
+# from https://raw.githubusercontent.com/sustainable-computing-io/kepler-model-server/main/tests/test_models/AbsComponentModelWeight/Full/KerasCompWeightFullPipeline/KerasCompWeightFullPipeline.json
+COPY --from=builder /opt/app-root/src/github.com/sustainable-computing-io/kepler/data/KerasCompWeightFullPipeline.json /var/lib/kepler/data/KerasCompWeightFullPipeline.json
```

Comment:
can you use `ADD https://raw.githubusercontent.com/sustainable-computing-io/kepler-model-server/main/tests/test_models/AbsComponentModelWeight/Full/KerasCompWeightFullPipeline/KerasCompWeightFullPipeline.json` instead of copying the file to `kepler` repo?

Reply:
done